docs/getting-started/delete-on-dockstore.rst

```diff
@@ -0,0 +1,44 @@
+Deleting an entry on Dockstore
+===================================
+
+Tutorial Goals
+--------------
+
+-  Deleting an entry
+-  Uninstalling the Dockstore GitHub App from the source GitHub repo
```

Comment:
Pondering if a more generic sentence should be used like "Ensuring that the entry doesn't re-appear" since below you mention that you could also remove the entry from the .dockstore.yml or delete the .dockstore.yml